cmd/identities/delete.go

```diff
@@ -0,0 +1,49 @@
+package identities
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ory/kratos/cmd/cliclient"
+	"github.com/ory/kratos/internal/httpclient/client/admin"
+)
+
+var deleteCmd = &cobra.Command{
+	Use:  "delete",
+	Args: cobra.MinimumNArgs(1),
```

Comment:
Could you add some helper descriptions to these commands? Examples could also be very helpful!